lib/Migration/Version0017Date20200202112903.php

```diff
@@ -90,6 +95,7 @@ public function changeSchema(IOutput $output, Closure $schemaClosure, array $opt
 			'instance', 'string', [
 						  'notnull' => false,
 						  'length'  => 255,
+						  'default' => $this->config->getSystemValue('instanceid'),
```

Comment:
```suggestion
						  'default' => '',
```

The instance id is definetly a wrong setting there. if something should be used, it's the url of the server which is used when doing federation. But I think it's not good to put that into the database structure

Reply:
Default must be an empty string. I'll check that later today

Reply:
I didn't found any references for the `instance` column in the code, only this https://github.com/nextcloud/circles/blob/v0.18.2/lib/Service/ConfigService.php#L430
I guess there are some missing parts to use this not null column...

Reply:
> 
> 
> Default must be an empty string. I'll check that later today

I see, but not null constraint with empty string isn't possible with PostgreSQL

Reply:
but not-null is false, so empty is allowed. But in that case it needs a default, which is missing here.

Reply:
Got it, I'll update this pull request